sentinel/pkg/checker/signer/utils.go

```diff
@@ -34,6 +34,30 @@ func ExtractExpirationFromContract(ctx context.Context, jsonrpc string, submissi
 	return &expirationDate, nil
 }
 
+func GetSignerAddresses(ctx context.Context, jsonrpc string, submissionProxy string) ([]string, error) {
```

Comment:
have you tried to call this function separately? if you didn't I recommend you to try test call this function from separate script